bucket/figma.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "101.6",
+    "description": "Figma connects everyone in the design process so teams can deliver better products, faster.",
```

Comment:
The description sounds like an advert. It doesn't help me understand what Figma does

Reply:
I guess it's all in there. "Figma somehow connects designers to develop better designs, faster". It's more than just a design tool, but maybe you can suggest a better one..

> PS: No affiliations to Figma, just a simple user here 👋🏻

Reply:
"A vector graphics editor and prototyping tool focusing on use in user interface and user experience design, with an emphasis on real-time collaboration"